passiveData/src/commonMain/kotlin/org/sagebionetworks/assessmentmodel/passivedata/asyncaction/AsyncActionStatus.kt

```diff
@@ -0,0 +1,46 @@
+package org.sagebionetworks.assessmentmodel.passivedata.asyncaction
+
+import org.sagebionetworks.assessmentmodel.passivedata.internal.StringEnum
+
+enum class AsyncActionStatus(val description: String) : StringEnum {
+    /// Initial state before the controller has been started.
+    IDLE("idle"),
```

Comment:
FYI: this enum doesn't need to be serializable so the string values don't need to match.